lib/java/test-utils/src/main/java/org/enso/test/utils/ProjectUtils.java

```diff
@@ -121,21 +119,6 @@ public static void testProjectRun(Path projDir, Consumer<Value> resultConsumer)
 
   /** Deletes provided directory recursively. */
   public static void deleteRecursively(Path rootDir) throws IOException {
-    Files.walkFileTree(
-        rootDir,
-        new SimpleFileVisitor<>() {
-          @Override
-          public FileVisitResult visitFile(Path file, BasicFileAttributes attrs)
-              throws IOException {
-            Files.delete(file);
-            return FileVisitResult.CONTINUE;
-          }
-
-          @Override
-          public FileVisitResult postVisitDirectory(Path dir, IOException exc) throws IOException {
-            Files.delete(dir);
-            return FileVisitResult.CONTINUE;
-          }
-        });
+    Files.walk(rootDir).sorted(Comparator.reverseOrder()).map(Path::toFile).forEach(File::delete);
```

Comment:
Collect all files that return `false` from `File.delete()`. Check that `rootDir.exists()` is false at the end. If not, print out all the files that refused to be deleted.